docs/standards/lsp-background/erc725.md

```diff
@@ -14,69 +13,78 @@ sidebar_position: 1
 
 ## Introduction
 
-Using Externally Owned Accounts (EOA) is limiting and risky as if the private key of the account gets leaked, anyone can execute any transaction from it, transferring all assets owned by the account. Also, data cannot be attached to it and it's limited to performing the CALL and CREATE opcode.
-
-In order to have a more secure account, more flexible execution on the blockchain and the ability to attach endless data, a smart contract should be used. It ensures a safe ownership process, more operations to execute and endless data store availability.
-
-
-## What does this standard represent?
+EOAs (Externally Owned Accounts) are controlled by private keys and can perform specific operations such as [**CALL**](https://www.evm.codes/#f1) to interact with other addresses and [**CREATE**](https://www.evm.codes/#f0) to create other smart contracts. However, they do not have any internal logic or storage capabilities.
 
-This standard is formed from 2 sub-standards that could be implemented stand-alone or together.   
-**[ERC173](https://eips.ethereum.org/EIPS/eip-173)** is used in both standards to provide ownership on the smart contract functions, where the owner could be an Externally Owned Account (EOA) or a smart contract (Another ERC725 contract, Key Manager contract, etc).
+Smart contracts, on the other hand, can be controlled by EOAs or other smart contracts, which provide a flexible ownership management mechanism. They can store data and perform more complex operations, making them useful for a wide range of use cases, including digital assets, decentralized applications, DAO and smart contract based accounts.
 
-![ERC725 Ownership](../../../static/img/standards/erc725/erc725-owner.jpeg)
+## ERC725X
 
-### ERC725X - Generic Executor
+There is a growing need for more operations beyond the basic **CALL** and **CREATE** functionality of an externally owned account (EOA). These two operations allows to interact with other addresses on the blockchain and perform tasks such as calling functions, transferring native tokens, and creating new contracts. As the number of use cases for smart contracts increases, so too does the need for additional functionality.
 
-This sub-standard enables the owner to execute generic calls on any other account, including calling contract functions and transferring native tokens along with the call. These actions are possible via the [`execute(...)`](../smart-contracts/lsp0-erc725-account.md#execute) function in the smart contract.
+One such operation is the ability to manipulate storage of a contract and execute other contracts logic in the caller context, which is not possible with EOA. Additionally, the ability to create contracts at predetermined addresses, as well as, performing calls without causing any state changes are also important features that are currently not available to EOAs.
```

Comment:
I wonder if discussing caller context in this example is going a bit too technical. Could just start super basic - EOAs don't have any data store which limits the functionality they can perform

Reply:
It's the motivation for `ERC725X`, but yeah ill try to simplify

Reply:
Ah okay well even going into more detail about what that means could be cool